### PR TITLE
➕ Add stylelint & stylelint-config-prettier

### PR DESCRIPTION
**Pull request by sweisz**
Add stylelint & prettier/stylelint-config-prettier

https://stylelint.io/user-guide/get-started
https://github.com/prettier/stylelint-config-prettier

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,9 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "prettify": "prettier --write ."
+    "prettify": "prettier --write .",
+    "stylelint": "stylelint \"**/*.css\"",
+    "stylelint-check": "stylelint-config-prettier-check"
```
Comment (phlgr) [inline package.json:11]: Not sure how useful this script is :)

**Review by phlgr — CHANGES_REQUESTED** [review]

**Review by lmachens — APPROVED** [review]
🚀

**Review by mrana17 — APPROVED** [review]
looks nice 🚀

**Review by christophvogel — APPROVED** [review]
GJ 👍

File: package.json (hunk 1 above)
Comment (sweisz) [reply package.json:11]: TX, i would like to use it.

**Pull request merged**